packages/amplify-e2e-core/src/categories/custom.ts

```diff
@@ -26,6 +26,9 @@ export const addCFNCustomResource = async (cwd: string, settings: any, testingWi
   if (settings.promptForCategorySelection) {
     chain.wait('Select the categories you want this custom resource to have access to').sendCtrlA().sendCarriageReturn();
   }
+  if (settings.promptForCustomResourcesSelection) {
+    chain.wait('Select the one you would like your custom resource to access').sendCtrlA().sendCarriageReturn();
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/5849952/225766810-93588ea8-326b-4915-b8ac-ae75dcf86a04.png)

Reply:
Oh this was getting stuck.